scripts/integration_trigger.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e -o pipefail
+
+HELM_BRANCH="${HELM_BRANCH:-master}"
+STATUS_URL="https://api.github.com/repos/NervanaSystems/${CIRCLE_PROJECT_REPONAME}/statuses/${CIRCLE_SHA1}"
+build=$(curl \
+  --header "Content-Type: application/json" \
+  --request POST https://circleci.com/api/v1.1/project/github/NervanaSystems/nervana-helm/tree/${HELM_BRANCH}?circle-token=${CIRCLE_TOKEN} \
+  --data @- <<EOF
+  {
+    "build_parameters": {
+      "CUSTOM_VALUES": "${CUSTOM_VALUES}",
+      "STATUS_URL": "${STATUS_URL}"
+    }
+  }
+EOF
+)
```

Comment:
If we install this: https://github.com/rockymadden/circleci-cli

This would simplify this entire thing to:
```
circleci trigger NervanaSystems/nervana-helm master -K CUSTOM_VALUES -V "${CUSTOM_VALUES}" -K STATUS_URL -V "${STATUS_URL}"
```

Reply:
One more thing, I'm not sure if some part of this should still be exposed to the .circleci/config.yml instead of being completely obfuscated in a single script. i.e.

```
command: |
   integration_trigger.sh
```

v.s.
```
command: |
  circleci trigger NervanaSystems/nervana-helm master -K CUSTOM_VALUES -V "${CUSTOM_VALUES}" -K STATUS_URL -V "${STATUS_URL}"
  github_status update "pending" "${STATUS_URL}"
```

and later on once the test completes:
```
command: |
  github_status update "success" "${STATUS_URL}"
```
```
command: |
  github_status update "failure" "${STATUS_URL}"
```